src/web/lib/braze/buildBrazeMessages.ts

```diff
@@ -0,0 +1,79 @@
+import {
+	hasCurrentBrazeUser,
+	setHasCurrentBrazeUser,
+	clearHasCurrentBrazeUser,
+} from '@root/src/web/lib/hasCurrentBrazeUser';
+import { initPerf } from '@root/src/web/browser/initPerf';
+import { record } from '@root/src/web/browser/ophan/ophan';
+import { BrazeMessages, BrazeMessagesInterface } from './BrazeMessages';
+import { checkBrazeDependencies } from './checkBrazeDependencies';
+import { getInitialisedAppboy, SDK_OPTIONS } from './initialiseAppboy';
+import { NullBrazeMessages } from './NullBrazeMessages';
+
+const maybeWipeUserData = async (
+	apiKey?: string,
+	brazeUuid?: null | string,
+): Promise<void> => {
+	if (apiKey && !brazeUuid && hasCurrentBrazeUser()) {
+		const appboy = await getInitialisedAppboy(apiKey);
+
+		try {
+			appboy.wipeData();
+			clearHasCurrentBrazeUser();
+		} catch (error) {
+			window.guardian.modules.sentry.reportError(error, 'braze-banner');
+		}
+	}
+};
+
+export const buildBrazeMessages = async (
+	isSignedIn: boolean,
+	idApiUrl: string,
+): Promise<BrazeMessagesInterface> => {
+	const dependenciesResult = await checkBrazeDependencies(
+		isSignedIn,
+		idApiUrl,
+	);
+
+	if (!dependenciesResult.isSuccessful) {
+		const { failure, data } = dependenciesResult;
+
+		if (SDK_OPTIONS.enableLogging) {
+			console.log(
+				`Not attempting to show Braze messages. Dependency ${failure.field} failed with ${failure.data}.`,
+			);
```

Comment:
Is this console test code?

Reply:
It's actually useful for local development. Since it'll only be output if that SDK_OPTIONS flag is set I think we'll leave it in (functionally this hasn't changed, just this line has moved).